examples/run-consensus-nodes.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
I suggest calling every node with the environment parameter that enables logging. E.g.
`RUST_LOG=hbbft=debug`. Otherwise there is no output.

Reply:
Done.